integration_tests/src/main/python/spark_session.py

```diff
@@ -83,6 +83,7 @@ def with_gpu_session(func, conf={}):
     """
     copy = dict(conf)
     copy['spark.rapids.sql.enabled'] = 'true'
+    copy['spark.rapids.memory.gpu.minAllocFraction'] = '0'
```

Comment:
nit, could we add a comment here (and the other place where this is set for the tests)? So we remember why this is needed.

Reply:
Done